src/components/ebay-video/component.js

```diff
@@ -115,10 +121,14 @@ module.exports = {
             this.state.action = input.action;
             this.takeAction();
         }
+        if (input.volumeSlider === false) {
```

Comment:
Can this be reduced to just `this.state.volumeSlider = input.volumeSlider`?

Reply:
We could, but then we still have to check if input.volumeSlider is undefined or if setting it to undefined puts the component in a weird state.... I think that would work though

Reply:
Ok, just leave as is for now then, to be safe.